### DISCUSSION TITLE
Chia wallet problem for help.

### DISCUSSION THREAD

**Discussion by abcc3** [Chia-Network/chia-blockchain#8610] [Support]
Hello, developer. I'm running Chia desktop wallet. My system is win10.

The space of C disk where my current computer system is located is limited.

How can I transfer Chia's blockchain data to another hard disk.

If you can, please give a tutorial, thank you.

Any help is welcome.

Let me ask another question. Do you add the function of automatically deleting useless blockchains in the wallet installation file of the new version? If not, please join.

**Comment by loppefaaret** [comment]
new updated versions of Chia will reuse the databases in use
To move the database, there are a couple of different ways you can try:
first is through the `config.yaml` file located in your users folder `~\.chia\mainnet\config\config.yaml`
in this file, under the `full_node:` section, you will find this:
```
  database_path: db/blockchain_v1_CHALLENGE.sqlite
```
changing that to something like `T:\chia\blockchain_v1_CHALLENGE.sqlite` would direct it to that folder instead


another option is to make use of the env var `CHIA_ROOT` and set that to point to `T:\chia` for instance - that would make chia use that as its working folder, and you could move the contents of `~\.chia\mainnet\` to `T:\chia\`

**Comment by BrandtH22** [comment]
@abcc3 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
